include/radix/data/model/MeshLoader.hpp

```diff
@@ -13,12 +13,72 @@ class Entity;
 
 class MeshLoader {
 public:
-  static Mesh& getMesh(const std::string &path);
+    /**
+     * @brief The VertexAttribInfo struct
+     * Holder for VertexAttrib data
+     */
+    struct VertexAttribInfo {
+        unsigned int index;    /** < attrib index */
+        int          channels; /** < number of channels of vertex */
+        unsigned int type;     /** < type vertex */
+        void         *offset;     /** < offset pointer */
+    };
+
+  /**
+   * @brief getMesh read mesh file from hard and upload to GPU
```

Comment:
@wow2006 I too don't understand this man :D
... read file from hard **drive** ... or something like that ???
and maybe spaced kind of out of the vertical line